### PR TITLE
tentacle_manager integration 

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Tentacles-Manager/pull/10

### CODE REVIEW

File: octobot_commons/config_manager.py [hunk 1]
```diff
@@ -130,21 +129,6 @@ def is_in_dev_mode(config):
     return CONFIG_DEBUG_OPTION in config and config[CONFIG_DEBUG_OPTION]
 
 
-def update_evaluator_config(to_update_data, current_config, deactivate_others):
```
Comment (Herklos) [inline octobot_commons/config_manager.py]: :+1:

File: octobot_commons/config_manager.py [hunk 2]
```diff
@@ -130,21 +129,6 @@ def is_in_dev_mode(config):
...
-    _update_activation_config(to_update_data,
-                              current_config,
-                              CONFIG_EVALUATOR_FILE_PATH,
-                              CONFIG_EVALUATOR_FILE,
-                              deactivate_others)
-
-
-def update_trading_config(to_update_data, current_config):
```
Comment (Herklos) [inline octobot_commons/config_manager.py]: :+1:

File: octobot_commons/config_manager.py [hunk 3]
```diff
@@ -289,51 +273,33 @@ def clear_dictionaries_by_keys(dict_dest, dict_src):
...
-def update_tentacle_config(klass, config_update):
-    current_config = klass.get_specific_config()
-    # only update values in config update not to erase values in root config (might not be editable)
-    for key, val in config_update.items():
-        current_config[key] = val
-    with open(klass.get_config_file_name(), "w+") as config_file_w:
-        config_file_w.write(dump_json(current_config))
-
-
-def factory_reset_tentacle_config(klass):
```
Comment (Herklos) [inline octobot_commons/config_manager.py]: :+1:

File: octobot_commons/config_manager.py [hunk 4]
```diff
@@ -289,51 +273,33 @@ def clear_dictionaries_by_keys(dict_dest, dict_src):
     return dict_dest
 
 
-def _update_activation_config(to_update_data, current_config, config_file_path, config_file, deactivate_others):
-    from octobot_commons.tentacles_management.class_inspector import get_class_from_string, evaluator_parent_inspection
+def update_activation_config(to_update_data, current_config, deactivate_others):
```
Comment (Herklos) [inline octobot_commons/config_manager.py]: I don't understand why this is not handled by the Tentacle Manager ?

File: octobot_commons/constants.py [hunk 5]
```diff
@@ -60,26 +60,7 @@
 START_PENDING_EVAL_NOTE = "0"
 
 # tentacles
-TENTACLE_CONFIG_FOLDER = "config"
 TENTACLE_DEFAULT_CONFIG = "default_config"
-TENTACLE_DEFAULT_FOLDER = "Default"
```
Comment (Herklos) [inline octobot_commons/constants.py]: :100:

File: octobot_commons/tentacles_management/abstract_tentacle.pxd [hunk 6]
```diff
@@ -21,24 +21,6 @@ cdef class AbstractTentacle:
 
     cpdef str get_name(cls)
 
-    cpdef str get_config_tentacle_type(cls)
-
-    cpdef str get_tentacle_folder(cls)
-
     cpdef list get_all_subclasses(cls)
 
-#     cpdef str get_config_folder(cls, str config_tentacle_type=*)
```
Comment (Herklos) [inline octobot_commons/tentacles_management/abstract_tentacle.pxd]: :+1:

File: octobot_commons/tentacles_management/abstract_tentacle.py [hunk 7]
```diff
@@ -61,57 +41,8 @@ def get_all_subclasses(cls) -> list:
                 subclasses_list += subclass.get_all_subclasses()
         return subclasses_list
 
-    @classmethod
```
Comment (Herklos) [inline octobot_commons/tentacles_management/abstract_tentacle.py]: :+1:

File: octobot_commons/tentacles_management/abstract_tentacle.py [hunk 8]
```diff
@@ -61,57 +41,8 @@ def get_all_subclasses(cls) -> list:
...
-            with open(cls.get_config_file_schema_name()) as f:
-                return f.read()
-        except Exception as e:
-            if raise_exception:
-                raise e
-        return ""
-
     @classmethod
     # returns DESCRIPTION class attribute, used as documentation
+    #TODO replace with markdown doc
```
Comment (Herklos) [inline octobot_commons/tentacles_management/abstract_tentacle.py]: It's a feature in development in Tentacle Manager ?

Comment (GuillaumeDSM) [reply octobot_commons/tentacles_management/abstract_tentacle.py]: In web interface yes

File: octobot_commons/config_manager.py (hunk 4 above)
Comment (GuillaumeDSM) [reply octobot_commons/config_manager.py]: Right, should be there, i'll change it

File: .travis.yml [hunk 9]
```diff
@@ -63,9 +63,9 @@ matrix:
...
+      secure: Z+/pkPOu5iJTBkuROtL5URWC9XQnV3j/8VpacsTjSRwBHJ6OO1aJrncYM1rUTLReZqXXkEnZ9WJY/9VOCMoAEApwvx1Qo8aYDfwSi5U0XALyqmncleFKDNj5uTGNIhPEB2y8W5jz9uh3tCFJvkVC4JbyqDQ/VoTJQtkYNDnhYNONoprRSntiQP+pwc9G2hVVFzUcH67oAe0RxYLBPEETNdIrNEj3Qcab3Bq/CoHYPTqSOnJPan1/+9+xux51ZRV9W1mn7ITjwFSkNm+7EyGK6iUBnTc248xRW9Z+36q71JSxe5QgV5ovdKBItuJqnS9VrZMkS0KWaB79pAUKFoXIyDvBG6n3V/5Iha0j3JX5CBZJI35RZ7wZJ5otfkQVajxKTeA9nRuiSTrZhZ35DaJdeZEhN4D59lD0Bz3h3srNJ1tACmNc7r0VLycCpDyyQM2x+TaWGlwoiwXrEMaQgyql+LuSqtJ5MKO1delkZqhJ/F+exICVRT7kDiiePQGBbJhXoh3ytdORGL2KIBxdqZBcIbpgs1crCp4uolQuE/IvdxqBnvjNIR9TQYGegAbv1x8GNpma5VgTTYp98MLinjiA/VMm2GSjJH1DLjNjbf+s3VsISrbeiYK3HGJdCszi+cUEVnERZleazmpzf1Y5a2NvoRQBh67LwWhDWgzrDTNMTLQ=
```
Comment (Herklos) [inline .travis.yml]: Don't forget to rebase please ;)

**Pull request merged**